src/hdmf/build/map.py

```diff
@@ -1480,6 +1502,9 @@ def get_container_cls(self, **kwargs):
             for k, field_spec in attr_names.items():
                 if not spec.is_inherited_spec(field_spec):
                     fields[k] = field_spec
+            d = self.__get_cls_dict(parent_cls, fields)
```

Comment:
What is the purpose of this line vs line 1509 below?

Reply:
@rly good catch! That was a merge mistake. Deleting that line solves the issue.